librz/include/rz_rop.h

```diff
@@ -42,6 +43,7 @@ typedef struct rz_rop_gadget_info_t {
 	ut64 curr_pc_val; ///< Current PC value.
 	bool is_pc_write; ///< PC write flag.
 	bool is_syscall; ///< Syscall flag.
+	HtUP *analysis_cache; ///< Analysis cache.
```

Comment:
```suggestion
	HtUP *analysis_cache; ///< Hitlist cache indexed by address for RzAnalysisOps. The analysis ops become part of a gadget.
```

Is this correct? Otherwise please describe what it is used for please.

Reply:
And add type annotation `/* <RzAnalysisOp> */`

Reply:
I have edited this